code/game/objects/items/debug_items.dm

```diff
@@ -288,16 +287,19 @@
 		TRAIT_MEDICAL_HUD,
 		TRAIT_SECURITY_HUD,
 		TRAIT_BARMASTER,
+		TRAIT_ALL_SURGERIES,
 		TRAIT_METALANGUAGE_KEY_ALLOWED
 	)
 
-/obj/item/debug/orb_of_power/pickup(mob/user)
+/obj/item/debug/orb_of_power/pickup(mob/living/user)
```

Comment:
Can not living mob pick this up?
Would be safer to just cast bellow

Reply:
Only living mobs have inventory

Reply:
well, changed anyway